packages/cli/src/util/link/repo.ts

```diff
@@ -124,22 +131,29 @@ export async function ensureRepoLink(
     if (!remoteUrls) {
       throw new Error('Could not determine Git remote URLs');
     }
-    const remoteNames = Object.keys(remoteUrls);
+    const remoteNames = Object.keys(remoteUrls).sort();
```

Comment:
Why sort? Is this handling the case when you have `origin` and `original` or something like that?

It would be great to add a test.

Reply:
I tinkered with this and I think it's fine. If there's a `'origin'` remote, then it forces the remote to be `'origin'`. The sort order doesn't matter.

Interestingly enough, apparently the default `.sort()` will sort capitalized words before lowercase words:

```
> ['a', 'B'].sort()
[ 'B', 'a' ]
```

Reply:
The default sort is basically comparing the unicode values of the chars.

You could handle cases by using `localeCompare()`.

```js
['a', 'B'].sort((a,b) => a.localeCompare(b)) // ['a', 'B']
```

Reply:
I guess I've been using `localeCompare()` for so many years, I rarely do a `.sort()` without it.